source/EngineTests/DescriptionEditTests.cpp

```diff
@@ -51,7 +51,8 @@ TEST_F(DescriptionEditTests, correctConnections)
 
 TEST_F(DescriptionEditTests, addThirdConnection1)
 {
-    auto data = CollectionDescription().addCells({
+    CollectionDescription data;
+    data.cells() = {
```

Comment:
use `auto data = CollectionDescription().cells({CellDescription()..., )`
check all your other changes for this. also in other files.